modules/openxAnalyticsAdapter.js

```diff
@@ -56,10 +57,10 @@ const DEFAULT_ANALYTICS_CONFIG = {
   enableV2: false,
   testPipeline: false,
   adIdKey: 'hb_adid',
-  utmTagData: {},
+  campaign: {},
   adUnits: [],
   payloadWaitTime: AUCTION_END_WAIT_TIME,
-  payloadWaitTimePadding: 100
+  payloadWaitTimePadding: 2000
```

Comment:
is `payloadWaitTimePadding` needed for utm ?

Reply:
No.  I realized that 100ms was too short of a time and the analytics adapter was sending incomplete payloads so I raised the default to 200